curated_transformers/tokenizers/hf_hub.py

```diff
@@ -38,6 +40,30 @@ def from_hf_hub_to_cache(
         """
         raise NotImplementedError
 
+    @classmethod
+    @abstractmethod
+    def from_fsspec(
+        cls: Type[SelfFromHFHub],
+        *,
+        fs: AbstractFileSystem,
+        fsspec_args: Optional[Dict[str, Any]] = None,
+        model_path: str,
```

Comment:
The order is swapped compared to the signature in the other definitions.

Reply:
Fixed.